emstorm

```diff
@@ -0,0 +1,183 @@
+# Ships
+
+ship "Electromagnetic Storm"
+	attributes
+		"hull" 100
+		"automaton" 1
+		"mass" 10
+		"drag" 1
+		"heat dissipation" .9
+		"outfit space" 1
+		"weapon capacity" 1
+		"energy generation" .7
+		"energy capacity" 100
+		"turn" 40
+		"turning energy" .2
+		"thrust" 10
+		"thrusting energy" .3
+	outfits
+		"Storm"
+		
+	engine -17 35
+	engine 17 35
+	turret 0 0 "Storm"
+	
+# Outfits
+
+outfit "Storm"
+	category "Turrets"
+	cost 0
+	thumbnail "outfit/unknown"
+	"mass" 1
+	"outfit space" -1
+	"weapon capacity" -1
+	"turret mounts" -1
+	weapon
+		"inaccuracy" 0
+		"turret turn" 10
+		"velocity" 200
+		"lifetime" 30
+		"reload" 1
+		"firing energy" 0
+		"firing heat" 0
+		"shield damage" 1
+		"hull damage" 0.1
+		"ion damage" 0.1
+		
+# Fleet
+fleet "Electromagnetic(weak)"
+	government "Korath Nanobots"
+	personality
+		heroic
+		vindictive
+	variant
+		"EMStorm" 8
+		
+fleet "Electromagnetic(moderate)"
+	government "Korath Nanobots"
+	personality
+		heroic
+		vindictive
+	variant
+		"EMStorm" 15
+		
+fleet "Electromagnetic(strong)"
+	government "Korath Nanobots"
+	personality
+		heroic
+		vindictive
+	variant
+		"EMStorm" 22
+		
+# Mission
+mission "Remnant: Stranded Korath"
+	landing
+	source
+		government "Remnant"
+	name "Korath ships stranded in Remnant Territory"
+	description "There are Korath fleets located in the Caeculus, Stercutus and Segesta systems. Find and eliminate them, then return to <planet> for payment."
+	to offer
+		has "event: remnant: void sprite research"
+		has "Remnant: Key Stones: done"
+		has "Remnant: Defense 3: done"
+		has "Remnant: Bounty: done" >= 3
+	waypoint Caeculus
+	waypoint Stercutus
+	waypoint Segesta
+	on offer
+		conversation
+			`As you land, you are greeted by a man and a woman, both dressed in quite a striking manner, perhaps as a sign of distinction. They collectively begin to chant in an almost gleeful tone`
```

Comment:
You are missing a **lot** of punctuation in your conversation text, particularly before character speech.